draft-ietf-core-dns-over-coap.md

```diff
@@ -391,7 +392,20 @@ A successful response:
       ;; ANSWER SECTION:
       ;example.org.         79689   IN      AAAA    2001:db8:1::1:2:3:4
 
-When a DNS error—NotImp (RCODE = 4) in response to a DNS Update (OPCODE = 5) for "example.org" in this case—is noted in the DNS response, the CoAP response still indicates success.
+When a DNS error—NxDomain (RCODE = 3) for "does.not.exist" in this case—is noted in the DNS response, the CoAP response still indicates success.
+
+    2.05 Content
+    Content-Format: application/dns-message
+    Payload (human-readable):
+    ;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 0
+    ;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0
+
+    ;; OPT PSEUDOSECTION:
+    ; EDNS: version: 0, flags:; udp: 0
```

Comment:
This looks like an EDNS(0) option per RFC 6891 is included with UDP size "0". That could be confusing; normally the client would have to indicate its supported UDP size?  Like 1024 can be typically achieved on all/most 6LoWPAN networks. Or maybe we shouldn't include this option for the example to avoid such details?

Reply:
Ah, sorry Copy pasta from me actually using dig to generate this example. Will remove!